packages/core/src/components/ui/ImageGallery/image-gallery-selector.module.scss

```diff
@@ -96,7 +96,7 @@
       color: var(--fs-color-text-display);
     }
 
-    span {
+    span[data-fs-icon] {
```

Comment:
Fixes this span style
<img width="1538" alt="Screenshot 2023-03-10 at 20 12 43" src="https://user-images.githubusercontent.com/11325562/224445164-0566233f-8814-41c8-83da-4529c0ee2ac7.png">

Reply:
can we just sort the order?

`[data-fs-icon]span`